### PR TITLE
Do not apply to the default namespace at Quickstart guide

### PR DESCRIPTION
**Pull request by nakabonne**
**What this PR does / why we need it**:
Some people don't like to have their default namespace's tainted.

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/content/en/docs/quickstart/_index.md [hunk 1]
```diff
@@ -26,15 +26,18 @@ cd manifests
 ### 2. Installing control plane
 
 ``` console
-helm install pipecd ./manifests/pipecd --values ./quickstart/control-plane-values.yaml
+kubectl create namespace pipecd
+helm install pipecd ./manifests/pipecd \
+  --values ./quickstart/control-plane-values.yaml \
+  --namespace pipecd
```
Comment (nghialv) [inline docs/content/en/docs/quickstart/_index.md:32]: nit: Let's move the `namespace` option to be the first one in order.

```
helm install pipecd ./manifests/pipecd \
  --namespace pipecd \
  --values ./quickstart/control-plane-values.yaml
```

Comment (nakabonne) [reply docs/content/en/docs/quickstart/_index.md:32]: Right, it's more conventional.

**Comment by nakabonne** [comment]
Applied.

**Comment by nghialv** [comment]
Thank you.
/approve

**Pull request merged**